cli_reference/get_started_cli.adoc

```diff
@@ -36,6 +36,13 @@ for instructions per your workstation's operating system.
 
 [[installing-the-cli]]
 == Installing the CLI
+You can access the *About* page on the web console
```

Comment:
s/access/download the CLI from/

As it's written, the sentence sounds like it's just saying you can access the page but not what to do there, or that the page itself may or may not be enabled by the admin.

Suggestion: Maybe also start this with something like, "The easiest way to download the CLI is by accessing..."